### PR TITLE
Fixed some convoluted texts in remote.md and fixed a remote driver bug

### PR DESCRIPTION
**Pull request by mavenugo**
Signed-off-by: Madhu Venugopal <madhu@docker.com>

### CODE REVIEW

**Comment by mavenugo** [comment]
@aboch Addressed your comment on `d.networkType` in this push.

File: drivers/remote/driver.go [hunk 1]
```diff
@@ -10,23 +10,24 @@ import (
...
 	d.callback = dc
-	return networkType, d
+	return d.networkType, d
 }
 
 // Internal Convenience method to register a remote driver.
 // The implementation of this method will change based on the dynamic driver registration design
 func (d *driver) registerRemoteDriver(networkType string) (driverapi.Driver, error) {
-	newDriver := &driver{}
+	newDriver := &driver{networkType: networkType}
```
Comment (aboch) [inline drivers/remote/driver.go]: Thanks this is the change I was asking for.

**Comment by mavenugo** [comment]
@aboch Good point. Remote is too generic and can cause confusion. Please propose a good alternative name. But, when we change the name, lets change in all the places (including the package name, etc..).
For this PR, lets keep it Remote.

**Comment by mrjana** [comment]
LGTM

**Pull request merged**